scripts/verify_local_audio_extract.py

```
@@ -0,0 +1,651 @@
+        )
+
+    def assert_no_audio_extract_task(self, path: str, settle_seconds: float) -> None:
+        """After `settle_seconds`, fail if a succeeded audio_extract_text exists for this path."""
+        path_lit = sql_string_literal(path)
+        query = (
+            "SELECT n.path, f.revision, t.task_id, t.task_type, t.status "
+            "FROM file_nodes n "
+            "JOIN files f ON f.file_id = n.file_id "
+            "LEFT JOIN semantic_tasks t "
+            "  ON t.resource_id = f.file_id AND t.resource_version = f.revision "
+            f"WHERE n.path = '{path_lit}'"
```

Ruff (0.15.9)

[error] Possible SQL injection vector through string-based query construction

(S608)